### PR TITLE
feat(ui): Add Autocomplete and ConnectedAutocomplete for single values

### PR DESCRIPTION
**Pull request by bdauria**
This adds Autocomplete field for single selection

Screenshot from 2020-04-23 08-26-43

### CODE REVIEW

File: src/InputField/Select/Autocomplete/Connected/index.tsx [hunk 1]
```diff
@@ -2,84 +2,88 @@ import * as React from 'react';
...
-    setOpen(true);
-  };
+    const doOpen = (): void => {
+      setOpen(true);
+    };
 
-  const close = (): void => {
-    setOpen(false);
-  };
+    const close = (): void => {
```
Comment (Thebarda) [inline src/InputField/Select/Autocomplete/Connected/index.tsx:58]: According to `doOpen` function maybe this one should be `doClose` ?

**Comment by Thebarda** [comment]
**Tested with latest commit**

Single autocomplete options

Capture d’écran (2)


Multiple autocomplete options

Capture d’écran (7)

Remark : Is it possible to reduce spacing between options in multiple autocomplete ? To align with single autocomplete options

**Comment by bdauria** [comment]
@Thebarda I've reduced the padding in the Multi Autocomplete:

Screenshot from 2020-04-24 18-28-31

**Comment by Thebarda** [comment]
Tested and LGTM

**Pull request merged**